File: 3 periodo/Programação orientada a objeto 1/Questões 01 slid 03/questão01.py
#slide 19
lista = []
par = []
impar = []

while True:
    while True:    
        try:
            num = int(input("Digite o numero inicial: "))
            break
        except:
            continue
    if num < 0:
        break
    elif num == 0:
        lista.append(num)
    elif num % 2 == 0:
        par.append(num)
    elif num % 2 != 0:
        impar.append(num)
par.sort()
impar.sort()
par.extend(lista)
par.extend(impar)
print(par)

File: 3 periodo/Estrutura de dados I/Atividade de revisão algo2/PARTE 2/questão01.c
#include <stdio.h>

int soma(int num){
    if(num == 0)
        return 0;
    else{
        return num + soma(num - 1);
    }
}

int main(){
    int num;

    printf("DIgite um numero: ");
    scanf("%d", &num);
    int resultado = soma(num);
    printf("SOMA: %d", resultado);

    return 0;
}